docs/standards/smart-contracts/lsp0-erc725-account.md

```diff
@@ -39,7 +39,7 @@ function supportsInterface(bytes4 interfaceId) public view returns (bool)
 constructor(address newOwner) ERC725(newOwner)
 ```
 
-Sets the **initial owner** of the contract and registers **[LSP0ERC725Account](./interface-ids.md)**, **[ERC1271](./interface-ids.md)** and **[LSP1UniversalReceiver InterfaceIds](./interface-ids.md)**.
+The function sets the **initial owner** of the contract and registers the **LSP0 ERC725Account**, **[ERC1271](https://eips.ethereum.org/EIPS/eip-1271)**, and **[LSP1 UniversalReceiver InterfaceIds](./interface-ids.md)**.
```

Comment:
Please re-add the previous links to "./interface-ids.md", as we need these links so that people can easily find the `bytes4` interface IDs of the _lsp-smart-contracts_